### PR TITLE
 feat(sms.senders): add column to display sender reason of reject

### PR DESCRIPTION
**Pull request by Steffy29**
Ref: UXCT-99

### CODE REVIEW

File: packages/manager/modules/sms/package.json [hunk 1]
```diff
@@ -30,7 +30,9 @@
     "moment": "^2.22.2"
   },
   "devDependencies": {
-    "@ovh-ux/component-rollup-config": "5.0.0-beta.5"
+    "@ovh-ux/component-rollup-config": "5.0.0-beta.5",
+    "bootstrap": "~3.3.7",
```
Comment (antleblanc) [inline packages/manager/modules/sms/package.json]: Theses `devDependencies` aren't linked to the new feature. It should be added in a separate commit.

File: packages/manager/modules/sms/package.json [hunk 2]
```diff
@@ -30,7 +30,9 @@
...
+    "ovh-ui-kit": "^2.23.1"
```
Comment (antleblanc) [inline packages/manager/modules/sms/package.json]: Thanks for adding the same version from the `ovh-manager-telecom`.

Just for your information:
```sh
$ npm info ovh-ui-kit | grep latest
latest: 2.25.0
```

Comment (Steffy29) [reply packages/manager/modules/sms/package.json]: Not in the develop branch version, no ?

**Comment by Steffy29** [comment]
@antleblanc I rebased the commit message.

**Comment by antleblanc** [comment]
@Steffy29 Some commits has been squashed on the `develop` branch. Could you please rebase yours? Thanks!

**Comment by Steffy29** [comment]
@antleblanc rebase done

**Comment by antleblanc** [comment]
@Steffy29 I'm not sure that you've rebase correctly.

Normally you should have these two commit only:

```txt
feat(sms.senders): add column to display sender reason of reject
build: add needed dependencies
```

**Comment by Steffy29** [comment]
OK. I tried many rebases yesterday and it was the best result I could have. I don't understand why I had all those commits in my PR.
